### DISCUSSION TITLE
Add void as distro to check updates widgets

### DISCUSSION THREAD

**Discussion by ebiscaia** [qtile/qtile#4361] [Ideas]
Hi guys,

I would like to see VoidLinux in the distro list for the check updates widget. The line that need to be include in the library file is (I did it manually):

```
"VoidLinux": ("xbps-install -nuMS", 0),
```

Thanks,

Eddie

**Comment by elParaguayo** [comment]
Thanks.  Do you want to submit a pull request with this change?

**Comment by ebiscaia** [comment]
I could but I have never submitted a pull request to somebody else's project (I did with some of my projects with branching for learning purposes). Do I need to fork qtile for that?

**Reply by elParaguayo** [reply]
Yes. Create a fork. Create a new branch with your changes in. Make a pull request from your fork to the main repo.
